services/account_type/resource_owner.yaml

```diff
@@ -66,6 +66,15 @@ Resources:
             Condition:
               StringEquals:
                 'sts:ExternalId': !Ref ExternalId
+      ManagedPolicyArns:
```

Comment:
Resource Owner needs these policies too

Reply:
one thing I just now realized is that we can solve the "too many policies" problem be simply being more granular with links.  In other words, provisioning for resource ingest, or guardduty ingest (when we have it), etc.  No changes needed here, but something to think about

Reply:
Great point. Completely agree.